README.md

```diff
@@ -47,7 +47,7 @@ digest
 ```
 
 ### Testing
-For developers: if you want to test the code, then clone the Github repo, open a R console, change the 
+For developers: if you want to test the code, then clone the Github repo, open a R console, change the
```

Comment:
@NickiShaw Here and elsewhere, is there a reason why trailing spaces for mid-sentence line breaks have been removed in the README? I prefer to leave the spaces there because the spaces would need to be present if the text were not word-wrapped. But let me know if you think otherwise.